app/javascript/hooks/queries/admin/roles/useRoles.jsx

```diff
@@ -2,9 +2,12 @@ import { useQuery } from 'react-query';
 import axios from '../../../../helpers/Axios';
 
 export default function useRoles(query) {
-  return useQuery(['getRoles', query], () => axios.get('/admin/roles.json', {
-    params: {
-      search: query,
-    },
-  }).then((resp) => resp.data.data));
+  const params = {
+    search: query,
```

Comment:
if we rename `query and input to search` it will further simplify this into `params = { search }` and add more consistency.